inst/json/graph_cases_averted_config.json

```diff
@@ -0,0 +1,121 @@
+                "colsminus": ["casesAvertedPer1000ErrorMinus"],
+                "visible": true,
+                "thickness": 1.5,
+                "width": 0,
+                "opacity": 1
+            }
+        },
+        {
+            "x": ["PBO"],
+            "id": "PBO",
+            "type": "bar",
+            "name": "Switch to PBO ITN",
+            "marker": {
+                "color": "aquamarine",
+                "opacity": 0.5
+            },
+            "error_y": {
+                "type": "data",
+                "cols": ["casesAvertedPer1000ErrorPlus"],
+                "colsminus": ["casesAvertedPer1000ErrorMinus"],
+                "visible": true,
+                "thickness": 1.5,
+                "width": 0,
+                "opacity": 1
+            }
+        },
+        {
+            "x": ["IRS"],
+            "id": "IRS",
+            "name": "Only IRS",
+            "type": "bar",
+            "marker": {
+                "color": "purple",
+                "opacity": 0.5
+            },
+            "error_y": {
+                "type": "data",
+                "cols": ["casesAvertedPer1000ErrorPlus"],
+                "colsminus": ["casesAvertedPer1000ErrorMinus"],
+                "visible": true,
+                "thickness": 1.5,
+                "width": 0,
+                "opacity": 1
+            }
+        },
+        {
+            "x": ["ITN + IRS"],
+            "id": "ITN+IRS",
+            "name": "Add IRS to standard ITN",
+            "type": "bar",
+            "marker": {
+                "color": "darkred",
+                "opacity": 0.5
+            },
+            "error_y": {
+                "type": "data",
+                "cols": ["casesAvertedPer1000ErrorPlus"],
+                "colsminus": ["casesAvertedPer1000ErrorMinus"],
+                "visible": true,
+                "thickness": 1.5,
+                "width": 0,
+                "opacity": 1
+            }
+        },
+        {
+            "x": ["PBO + IRS"],
+            "id": "PBO+IRS",
+            "name": "Add IRS to PBO ITN",
+            "type": "bar",
+            "marker": {
+                "color": "orange",
+                "opacity": 0.5
+            },
+            "error_y": {
+                "type": "data",
+                "cols": ["casesAvertedPer1000ErrorPlus"],
+                "colsminus": ["casesAvertedPer1000ErrorMinus"],
+                "visible": true,
+                "thickness": 1.5,
+                "width": 0,
+                "opacity": 1
+            }
+        }
+    ],
+    "layout": {
+        "title": "Clinical cases averted per 1,000 people per year",
+        "yaxis": {
+            "title": "cases averted",
+            "showline": true,
+            "autorange": true
+        },
+        "xaxis": {
+            "title": "intervention",
+            "showline": true,
+            "tickvals": ["ITN", "PBO", "IRS", "ITN + IRS", "PBO + IRS"],
```

Comment:
These intervention names look different to those used in the prevalence graph. We should standardise these somewhere as I feel we're getting a lot of duplication though the code and json here